docs/capabilities/embeddings.mdx

```diff
@@ -13,6 +13,24 @@ Embeddings turn text into numeric vectors you can store in a vector database, se
 
 ## Generate embeddings
 
+### CLI
```

Comment:
Great addition! Let's add this to a CLI tab similar to other pages similar to the quickstart: https://docs.ollama.com/quickstart